docs/docs/agents/models.md

```diff
@@ -74,6 +74,39 @@ agent = create_react_agent(
 
     The example above uses `ChatAnthropic`, which is already supported by `init_chat_model`. This pattern is shown to illustrate how to manually instantiate a model not available through init_chat_model.
 
+## Adding model fallbacks
+
+In case of errors from LLM providers, you can add a fallback to a different model or a different LLM provider using `model.with_fallbacks([...])`:
```

Comment:
https://python.langchain.com/docs/how_to/fallbacks/#fallback-to-better-model

Reply:
We can check whether it works w/ streaming correctly

Reply:
checked streaming, works well!